web-client/iron-remote-desktop-rdp/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "@devolutions/iron-remote-desktop-rdp",
+  "author": "Nicolas Girot",
+  "email": "ngirot@devolutions.net",
+  "contributors": [
+    "Benoit Cortier",
+    "Irving Ou",
+    "Vladislav Nikonov",
+    "Zacharia Ellaham"
+  ],
```

Comment:
note: Feel free to add your name to the list of contributors.

Reply:
Done